app/controllers/series_controller.rb

```diff
@@ -136,7 +136,7 @@ def update
   def update_positions
     if params[:serial_works]
       @series = Series.find(params[:id])
-      @series.reorder(params[:serial_works])
+      @series.reorder(params[:serial_works].map { |s| s.to_i.to_s })
```

Comment:
Another call to `reorder` which is confusing Brakeman, and could be renamed to `reorder_works` instead.